pkg/kubernetes/testdata/deployment.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Hopefully having json fixture files can make them easier for people to read and modify!

Reply:
👍